packages/bridge-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Bump `@metamask/assets-controllers` to `^68.2.0` ([#5978](https://github.com/MetaMask/core/pull/5978))
+
```

Comment:
Ah, I think the changelog checker doesn't know that only `devDependencies` was bumped here, so you can drop this. I guess for now you can get around this by adding the `no-changelog` label.

Reply:
Okay gotcha!

Do we only add the changelog if a dev-dependency is a major version?

Reply:
Actually in this case the peer dependency major bump would cause this to change.

Reply:
Fixed in https://github.com/MetaMask/core/pull/5978/commits/356157b883037ef5bb12bea7a0aee14681bfb58b

Reply:
Yes, exactly, we don't ever document changes to dev dependencies, howver, if we were bumping `assets-controllers` by a major, then the peer dep would change, and we would document _that_.